ambry-named-mysql/src/main/java/com/github/ambry/named/MySqlNamedBlobDb.java

```diff
@@ -597,11 +617,17 @@ private NamedBlobRecord run_get_v2(String accountName, String containerName, Str
   private Page<NamedBlobRecord> run_list_v2(String accountName, String containerName, String blobNamePrefix,
       String pageToken, short accountId, short containerId, Connection connection) throws Exception {
     String query = "";
-    try (PreparedStatement statement = connection.prepareStatement(LIST_QUERY_V2)) {
+    try (PreparedStatement statement = blobNamePrefix == null ? connection.prepareStatement(LIST_ALL_QUERY_V2)
```

Comment:
Let's do this operation on strings, not PreparedStatement. Something like.
```
String queryStatement = blobNamePrefix == null? LIST_ALL_QUERY_V2: LIST_QUERY_V2;
try (PreparedStatement state = connection.prepareStatement(queryStatement)) {
}
```

Reply:
Sure, updated.